{
  "id": "CVE-2009-0819",
  "package_name": "mysql",
  "fix_state": "unknown",
  "cpes": [
    "cpe:2.3:a:mysql:mysql:5.1.23:*:*:*:*:*:*:*",
    "cpe:2.3:a:mysql:mysql:5.1.31:*:*:*:*:*:*:*",
    "cpe:2.3:a:mysql:mysql:6.0.9:*:*:*:*:*:*:*",
    "cpe:2.3:a:mysql:mysql:6.0.10-bzr:*:*:*:*:*:*:*",
    "cpe:2.3:a:mysql:mysql:*:*:*:*:*:*:*:*"
  ],
  "version_constraint": "<= 5.1.32-bzr || = 5.1.23 || = 5.1.31 || = 6.0.9 || = 6.0.10-bzr",
  "fixed_in_versions": "null",
  "severity": "Medium",
  "description": "sql/item_xmlfunc.cc in MySQL 5.1 before 5.1.32 and 6.0 before 6.0.10 allows remote authenticated users to cause a denial of service (crash) via \"an XPath expression employing a scalar expression as a FilterExpr with ExtractValue() or UpdateXML(),\" which triggers an assertion failure.",
  "urls": [
    "http://bugs.mysql.com/bug.php?id=42495",
    "http://dev.mysql.com/doc/refman/5.1/en/news-5-1-32.html",
    "http://dev.mysql.com/doc/refman/6.0/en/news-6-0-10.html",
    "http://www.securityfocus.com/bid/33972",
    "http://www.securitytracker.com/id?1021786",
    "http://www.vupen.com/english/advisories/2009/0594",
    "https://exchange.xforce.ibmcloud.com/vulnerabilities/49050",
    "https://oval.cisecurity.org/repository/search/definition/oval%3Aorg.mitre.oval%3Adef%3A7544"
  ]
}
